WcaOnRails/.rubocop.yml

```diff
@@ -54,3 +54,9 @@ Style/SignalException:
 Style/ClassVars:
   Exclude:
     - app/models/country.rb
+
+Style/IfInsideElse:
+  Enabled: false
+
+Lint/NonLocalExitFromIterator:
+  Enabled: false
```

Comment:
WOW. Ok, I *really* like @vincentwoo's solution. I think it follows very clearly from the problem definition.